### ISSUE TITLE
Production deployment: request for change in relative urls

### ISSUE THREAD

**Issue by mrtnrey** [inbo/crow#130]
@niconoe @peterdesmet 
While deploying on meteo.be, I received a small request for a change in the urls:

> Can you ask to the developers to make the change for relative urls:

```
[extraweb@intrarmi] /web/extraweb/www/birddetection.git]$ git diff vue.config.js
diff --git a/vue.config.js b/vue.config.js
index 343b112..bad3dd2 100644
--- a/vue.config.js
+++ b/vue.config.js
@@ -6,7 +6,7 @@ const commitHash = require("child_process")
   .toString();

 module.exports = {
- -   publicPath: process.env.NODE_ENV === "production" ? "/crow/" : "/",
+  publicPath: process.env.NODE_ENV === "production" ? "" : "/",
   configureWebpack: {
     plugins: [
       new webpack.DefinePlugin({

```

**Comment by niconoe** [comment]
Thanks for the report, @mrtnrey!

I've now solved this by making the URL prefix configurable via a `URL_PREFIX` environment variable that can be set in configuration files (we needed different values for the KMI and GitHub pages deployments). There's a default value of `""` for this variable, so you don't need to create any extra config file for your environment.

Can you download the [release 1.0.1](https://github.com/inbo/crow/releases/tag/v1.0.1), build it and make sure it works at the final URL. Special attention should go to the following points:

- Are the CSS/JS assets properly loaded? (that'll be obvious if not)
- Is the favicon properly loaded/displayed?
- Are the URLs generated by the "Copy link" button correct?

Excited to finally see this application in the wild very soon!

**Comment by mrtnrey** [comment]
Thanks!
You can test it yourself now:
https://www.meteo.be/birddetection
https://www.meteo.be/vogeldetectie
https://www.meteo.be/detectiondoiseaux

- Unfortunately we're in trouble with the "#" in the "Copy link" button.
- The favicon is still the INBO one (not blocking for me at least)

**Comment by niconoe** [comment]
About the "Copy link" issue:

can you try by creating an `.env.local` file in the root directory with content `URL_PREFIX="/services/birdDetection/"` before compiling the code (`npm run build`)?

Thanks!

**Comment by peterdesmet** [comment]
@mrtnrey the favicon can be changed in two ways:

- By replacing `public/favicon.ico` after pulling the code, but before running `npm run build`
- By replacing `favicon.ico` in the generated code after running `npm run build`

**Comment by mrtnrey** [comment]
Hi, deployment on meteo.be is refreshed: favicon is changed and links seem to work now!

If you should encounter anything else, please let me know asap.
We're reaching launch-day ;)

**Comment by peterdesmet** [comment]
Great! Just FYI, you cannot handcraft a URL from the custom domain and a query string:

https://www.meteo.be/vogeldetectie/#/?radar=behel&date=2021-03-04&interval=24&timedisplay=radarLocal&vpiMode=mtr&vpColorScheme=viridis

... since the redirect from https://www.meteo.be/vogeldetectie/ to https://www.meteo.be/services/birdDetection/#/?lang=nl ignores any other values in the query string. **That is fine**, since URLs created by the `Copy link` button always start with `https://www.meteo.be/services/birdDetection/`.

In other words, the following links:

https://www.meteo.be/birddetection
https://www.meteo.be/vogeldetectie
https://www.meteo.be/detectiondoiseaux

Should only be used to point to the default CROW page in that language.
